tests/integration/security/rbac/groupv1/rbac_group_v1_test.go

```diff
@@ -0,0 +1,125 @@
+// Copyright 2019 Istio Authors
```

Comment:
Can we change the path and filename to 
`tests/integration/security/rbac/v1/group/group_test.go`

And also change the `tests/integration/security/rbac/v2/rbac_v2_test.go` to `tests/integration/security/rbac/v2/basic/basic_test.go` 

In the future, you can just add directory `group` under `v2` for the v2 tests.

Reply:
Done.